### DISCUSSION TITLE
Change Color of the bar

### DISCUSSION THREAD

**Discussion by vishalkrsharma** [i3/i3#6119] [Q&A]
I want to change the color of the top bar to dark. How can i do this?

![image](https://github.com/i3/i3/assets/32286936/f9df92a3-9e07-48b2-83a7-8af5c96ca343)

**Comment by okraits** [comment]
If this is i3bar, then here's the relevant documentation: https://i3wm.org/docs/userguide.html#_colors

**Reply by vishalkrsharma** [reply]
No it is vscode's menu bar.
